project/forms.py

```diff
@@ -120,13 +149,14 @@ def _get_status_choices(self, status):
             return pre_approved_options
 
     def save(self, commit=True):
-        project = super(ProjectAdminForm, self).save(commit=False)
-        project.previous_status = self.initial_status
-        if self.initial_status != project.status:
-            update_openldap_project(project)
+        allocation = super(SystemAllocationRequestAdminForm, self).save(commit=False)
+        allocation.previous_status = self.initial_status
+        if self.initial_status != allocation.status:
+            # TODO: Check if there is another open allocation
```

Comment:
TODO?

Reply:
An offline comment I should have turned into an issue. Sorry about that.